.circleci/config.yml

```diff
@@ -1,6 +1,6 @@
 version: 2.1
 
-supported-eslint-versions: &supported-eslint-versions ['local', '7']
+supported-eslint-versions: &supported-eslint-versions ['local', '9']
```

Comment:
We do not need this logic anymore. We can just use the local ESLint version, since it's assumed to be v9.